### ISSUE TITLE
IdsAlert: A11Y: Information is conveyed in the form of symbol.

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise-wc#1024]
**Describe the bug**
Information is conveyed in the form of symbol..

**To Reproduce**

WCAG Guidelines: 1.3.3 Sensory Characteristics

1. Launch the URL: https://main.wc.design.infor.com/ids-alert/example.html
2. Navigate to different symbols present on the page and verify

Actual results:
Information is conveyed in the form of symbol. Different symbols are used to declare different information. But meaning of symbols a not written anywhere.

**Expected behavior**
Information should not be conveyed in the form of symbol. If it is conveyed then it should be written in textual form.

Instances/ Observation: Instances:
1. Alert> Side by Side

User Impact: User will not be able to identify the purpose of that symbol.

Dev Recommendation: Information about symbols should be present in textual form.

**Version**

- ids-enterprise: [e.g. v4.9.0 or v4.10.0]

**Screenshots**
https://www.screencast.com/t/yFzFi2sTHPFF

**Platform**
 - Windows 10
 - Browser Name: [Chrome

**Additional context**
None

**Comment by tmcconechy** [comment]
I guess we can show them on this page with text next to them or something. Usually they are not used alone like this in the example. But fair point 👍🏻
